scenarios/30_Final.cfg

```diff
@@ -159,16 +159,24 @@
         #po: NORMAL difficulty:
         {MSG_narrator _"Gold: (normal level): $ano_gawen_gold golden pieces, $level_point points"}
 #else
+#ifdef HARD
         {CLEAR_VARIABLE level_point}
         {VARIABLE level_point $ano_gawen_gold}
         {VARIABLE_OP level_point multiply 0.005}
         #po: HARD difficulty:
         {MSG_narrator _"Gold: (hard level): $ano_gawen_gold golden pieces, $level_point points"}
+#else
+        {CLEAR_VARIABLE level_point}
+        {VARIABLE level_point $ano_gawen_gold}
+        {VARIABLE_OP level_point multiply 0.007}
+        #po: NIGHTMARE difficulty:
+        {MSG_narrator _"Gold: (nightmare level): $ano_gawen_gold golden pieces, $level_point points"}
+#endif
 #endif
 #endif
 
         #po: ends in a colon because it leads into the following messages:
-        {MSG_narrator _"Extra Credit (5 points possible; 6 if you are playing on the 'hard' level): "}
+        {MSG_narrator _"Extra Credit (5 points possible; 6 if you are playing on the 'nightmare' level): "}
```

Comment:
Maybe 1 extra point for hard and 2 points for nightmare?

Reply:
ok *now* it should be done as of [6902fdf](https://github.com/nemaara/A_New_Order/pull/53/commits/6902fdf7e47d746bbef2ebf467979ed7983eee35).